src/components/Button.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { ButtonHTMLAttributes } from "react";
+import styled, { css } from "styled-components";
+
+export interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
```

Comment:
```suggestion
export interface ButtonProps extends ComponentProps<'button'> {
```
이러면 더 간단하게 표현할 수 있답니다!